guides/source/testing.md

```diff
@@ -1818,9 +1861,12 @@ test "renders a link to itself" do
 end
 ```
 
-If your application uses Ruby >= 3.0 or higher, depends on [Nokogiri >= 1.14.0](https://github.com/sparklemotion/nokogiri/releases/tag/v1.14.0) or
-higher, and depends on [Minitest >= >5.18.0](https://github.com/minitest/minitest/blob/v5.18.0/History.rdoc#5180--2023-03-04-),
-`document_root_element` supports [Ruby's Pattern Matching](https://docs.ruby-lang.org/en/master/syntax/pattern_matching_rdoc.html):
+If your application uses Ruby >= 3.0 or higher, depends on [Nokogiri >=
+1.14.0](https://github.com/sparklemotion/nokogiri/releases/tag/v1.14.0) or
+higher, and depends on [Minitest >=
```

Comment:
A Rails 7.2 application requires `Ruby >= 3.0` and `Nokogiri >= 1.14.0` already includes the "or higher"
```suggestion
If depends on [Nokogiri >=
1.14.0](https://github.com/sparklemotion/nokogiri/releases/tag/v1.14.0), and depends on [Minitest >=
```

Reply:
Is that rewording missing "if your application..."?